tools/containerize_gcsfuse_docker/Dockerfile

```diff
@@ -49,8 +49,8 @@ RUN go get -d ${GCSFUSE_REPO}
 
 WORKDIR ${GCSFUSE_PATH}
 # Branch name for building through a particular branch.
-ARG BRANCH_NAME="v${GCSFUSE_VERSION}"
```

Comment:
why is this code changed?

Reply:
Changed it to run integration tests on louhi. Reverted the change.